gitflow_toolbox/common/get_env.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+
+def get_env(*argv: str) -> str:
+    """Returns the first env var value found in the given list of env var names.
+
+    Args:
+        argv (str): list of env vars to search for
+
+    Raises:
+        Exception: if no value is found
+
+    Returns:
+        str: env var value
+    """
+    for arg in argv:
+        val = os.getenv(arg)
+        if val is not None:
+            return val
+    if len(argv) == 1:
+        raise Exception("env var " + str(argv[0]) + " is not set")
```

Comment:
You might use f-string instead

```suggestion
        raise Exception(f"env var {argv[0]} is not set")
```